learn/advanced/updating.md

```diff
@@ -1,27 +1,25 @@
 # Update to the latest Meilisearch version
 
-Currently, Meilisearch databases can only be opened by the Meilisearch version you used to create them. The following guide will walk you through all the steps to migrate an existing database from an older version of Meilisearch to the most recent one.
-
-If you already have a Meilisearch database with some data you don’t want to lose, you are in the right place!
-
 ::: danger
 This guide does not work for versions below v0.15. For more information, [contact support](https://discord.gg/meilisearch).
 :::
```

Comment:
>we may be able to add other version specific warnings to this section

Won't that get too overwhelming? Also, when the users start following the guide, they'll have to scroll up to this section to see if anything applies to them, which will be annoying ig. I think we should version-specific warnings under the step they apply to. But instead of using notes/warnings, we could add an H3 that says "For v0.x.y or below".